<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoriesWindow</class>
 <widget class="QMainWindow" name="StoriesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StatusWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>455</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>720</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgb(251, 253, 246);
    border: none;
}

QScrollBar:vertical {
    border: none;
    background: #f0f0f0;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #888;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
    subcontrol-origin: margin;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>438</width>
       <height>703</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border: none;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>17</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #000;
padding-left: 5px;</string>
           </property>
           <property name="text">
            <string>Stories</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addStoryButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: none;
image: url(:/icons/plus.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: none;
image: url(:/icons/close.png);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="myStoriesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>411</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>411</width>
          <height>91</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Product Sans Medium</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <fontweight>Medium</fontweight>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E7F1E6;
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000
</string>
        </property>
        <property name="text">
         <string>My stories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recentStoriesLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>136</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Product Sans Medium</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(251, 253, 246);
color: rgb(66, 73, 65);
padding-left: 5px;</string>
        </property>
        <property name="text">
         <string>Recent stories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="stories" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>412</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <widget class="QPushButton" name="friendStory_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <fontweight>Medium</fontweight>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #FFF;
font: 500 18pt &quot;Product Sans Medium&quot;;
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000;
</string>
           </property>
           <property name="text">
            <string>Friend Name</string>
           </property>
           <property name="iconSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="friendStory_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <fontweight>Medium</fontweight>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(251, 253, 246);
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000;
</string>
           </property>
           <property name="text">
            <string>Friend Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="friendStory_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <fontweight>Medium</fontweight>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(251, 253, 246);
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000;
</string>
           </property>
           <property name="text">
            <string>Friend Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="friendStory_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <fontweight>Medium</fontweight>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(251, 253, 246);
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000;
</string>
           </property>
           <property name="text">
            <string>Friend Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="friendStory_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>401</width>
             <height>81</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <fontweight>Medium</fontweight>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(251, 253, 246);
border: none;
border-radius: 30px;
text-align: left;
padding-left: 80px;
padding-bottom: 20px;
color: #000;
</string>
           </property>
           <property name="text">
            <string>Friend Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="noStoryLabel">
    <property name="geometry">
     <rect>
      <x>885</x>
      <y>320</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/unhappy.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="noStoryText">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>403</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans Medium</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(163, 163, 163);</string>
    </property>
    <property name="text">
     <string>No stories found</string>
    </property>
   </widget>
   <widget class="QWidget" name="storyContentBox" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>570</width>
      <height>726</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777196</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="storyTimeBox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>552</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>552</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(174, 209, 172);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(211, 232, 210);
border-radius: 50%;  </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="storyHBox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>552</width>
         <height>641</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>552</width>
         <height>726</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border: none;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="previousStoryButton">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/icons/previousButton.png);
background-color: transparent;
border: none;
color: rgba(255, 255, 255, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <widget class="QLabel" name="storyScreen">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>10</y>
             <width>420</width>
             <height>640</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>640</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>637</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans Medium</family>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 30px;
background-color: rgb(0, 0, 0);
color: #FFF;</string>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../Downloads/f826bf2ad330de7ee18d2bd8cd03b9ad.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
          <widget class="QWidget" name="expandinglabel" native="true">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>520</y>
             <width>420</width>
             <height>41</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(0, 0, 0, 128);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="mediaCaption">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>420</width>
                <height>41</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Product Sans Medium</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>sadsa</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextStoryButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/icons/nextButton.png);
background-color: transparent;
border: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>noStoryText</zorder>
   <zorder>noStoryLabel</zorder>
   <zorder>scrollArea</zorder>
   <zorder>storyContentBox</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
